### PR TITLE
Update artifact engine version and add telemetry.

### PR DESCRIPTION
**Pull request by omeshp**

### CODE REVIEW

File: Tasks/DownloadBuildArtifacts/task.loc.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "version": {
     "Major": 0,
     "Minor": 1,
```
Comment (GitHubSriramB) [inline Tasks/DownloadBuildArtifacts/task.loc.json]: should we also switch this task to use minor=milestone (126)?

File: Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts [hunk 2]
```diff
@@ -16,6 +16,10 @@ import { AzureStorageArtifactDownloader } from "./AzureStorageArtifacts/AzureSto
 import { ArtifactDetailsDownloader } from "./ArtifactDetails/ArtifactDetailsDownloader";
 import { JenkinsRestClient, JenkinsJobDetails } from "./ArtifactDetails/JenkinsRestClient"
 
+var packagejson = require('./package.json');
+
+const area: string = 'jenkinsdownloadartifacts';
```
Comment (GitHubSriramB) [inline Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts]: JenkinsDownloadArtifacts to be consistent with the other task?

File: Tasks/DownloadBuildArtifacts/task.loc.json (hunk 1 above)
Comment (omeshp) [reply Tasks/DownloadBuildArtifacts/task.loc.json]: not sure about impact, will take later.

File: Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts (hunk 2 above)
Comment (omeshp) [reply Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts]: done

**Pull request merged**